packages/moonstone/styles/internal/fonts.less

```diff
@@ -41,13 +40,15 @@
 	he: local("LG Smart UI AR HE TH") normal 300 600 700;
 	hi: local("LG Smart UI Devanagari");
 	kn: local("LG Smart UI Kannada");
+	ja: local("LG Smart UI JP") normal 300 600 700;
 	ml: local("LG Smart UI Malayalam") normal 300 600 700;
 	mr: local("LG Smart UI Devanagari"); // same as `hi`
 	or: local("LG Smart UI Oriya");
 	pa: local("LG Smart UI Gurmukhi") normal 300 600 700;
 	ta: local("LG Smart UI Tamil") normal 300 600 700;
 	te: local("LG Smart UI Telugu");
 	th: local("LG Smart UI AR HE TH") normal 300 600 700;
+	ur: local("LG Smart UI_Urdu") normal 300 600 700;
```

Comment:
It doesn't work. We need to remove `_` (`LG Smart UI Urdu`)

Reply:
I believe you must have a different set of fonts from me. 

From my PR comment:
> * The Urdu font name was updated as it didn't match the metadata in the font, which is unique in the font collection, as its family name (LG Smart UI Urdu), full name (LG Smart UI_Urdu), and postscript names (LGSmartUIUrdu) are all different. This requires special handling to be supported universally. Surprisingly this and the AR HE TH font is the only case of mismatch in the font collection.

I'll download and flash the latest TV build and extract the fonts from it so I'll be guaranteed to be working with the latest set of fonts.

Reply:
I've done some extensive testing and it appears the current version of blink on the Reference Platform build (72) no longer uses the full-name but now prefers the family-name in the metadata. I've removed the `_`, as you suggested, and it works as expected now on the TV. Also, I've removed the support for the `"Moonstone ur"` font-family, since it was causing the Urdu font to be used with all Arabic text, which is not what we want. Urdu font should only be used on Urdu/Arabic text when in the Urdu locale. It, like Japanese is technically a different typeface, so it shouldn't build onto the "LG Smart UI" -> "Moonstone" font-family.

I also removed the `normal` keyword from the non-Latin area as it didn't seem to actually be necessary for the fonts to apply, once this collision was removed.

PR has been updated.

Reply:
Good, please test on actual environment before comment.